Flixter/GridViewController.m

```diff
@@ -0,0 +1,108 @@
+//
+//  CollectionsViewController.m
+//  Flixter
+//
+//  Created by Angelina Zhang on 6/17/22.
+//
+
+#import "GridViewController.h"
+#import "MoviePosterCollectionCell.h"
+#import "DetailsViewController.h"
+#import "UIImageView+AFNetworking.h"
+
+@interface GridViewController () <UICollectionViewDataSource, UICollectionViewDelegate, UICollectionViewDelegateFlowLayout>
+
+@property (nonatomic, strong) NSArray *movieData;
+@property (weak, nonatomic) IBOutlet UICollectionView *moviesCollectionView;
+@property (weak, nonatomic) IBOutlet UIActivityIndicatorView *loadingIndicator;
+
+@end
+
+@implementation GridViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
```

Comment:
I like to put an extra line after super calls. Makes it obvious when you've forgotten them.